examples/mjpeg_server.py

```diff
@@ -85,7 +85,7 @@ class StreamingServer(socketserver.ThreadingMixIn, server.HTTPServer):
 picam2.start_preview()
 picam2.configure(picam2.video_configuration(main={"size": (640, 480)}))
 output = StreamingOutput()
-picam2.start_recording(JpegEncoder(), output)
+picam2.start_recording(JpegEncoder(), FileOutput(output))
```

Comment:
Nothing that needs changing, but a question of style for our users. Do you think they'd do better to create their own regular output class (`StreamingOutput` here) and wrap that in a `FileOutput`, or should we advise them to derive `StreamingOutput` directly from our base output class?

But let's leave this for now. If we think the alternative style is nicer we can change it later.